dm/task-configuration-file-full.md

```diff
@@ -160,6 +160,18 @@ loaders:
     # If checksum fails, the import is abnormal, which means the data is inconsistent or lost.
     # Therefore, it is recommended to always enable checksum.
     checksum-physical: "required"
+    # Only available for physical import. Specifies whether to perform the `ANALYZE TABLE <table>` for each table after CHECKSUM has finished.
+    # - "required" (default). Indicates that the Analyze operation will be performed after the import is complete, and if the analysis fails it will cause the task to pause and require manual processing by the user.
+    # - "optional". Indicates that the data will be analysed after the import is complete. If the analysis fails a warn log will be printed and the task will not be paused.
+    # - "off". Indicates that no data analysis will be performed after the import is complete.
+    # Analyze only affects statistics data and it is recommended that Analyze is set to off on in most scenarios.
+    analyze: "off"
+    # Only available for physical import. The concurrency of sending KVs data to TiKV. This can be increased when the direct network transfer speed between dm-worker and TiKV exceeds 10,000 Mb/s.
+    # range-concurrency: 16
+    # Physical Import Mode Whether to enable compression when sending KVs data to TiKV. Currently only Gzip compression is supported, either "gzip" or "gz" can be used. Compression is not enabled by default.
+    # compress-kv-pairs: ""
+    # pd-server address, just fill in one. If empty, the default is to use the pd address information from the TiDB query.
```

Comment:
```suggestion
    # One of the PD server addresses. If no address is specified, use the PD address information from the TiDB query by default.

Reply:
`One of the PD server addresses.` is not suitable, it means we can just fill in one. But filling more pd-addr is also okay.

Reply:
How about "One or more PD server addresses."

Reply:
LGTM